lib/config.js

```diff
@@ -13,6 +13,14 @@ config.messages.endpoints.messages = '/messages';
 config.messages.endpoints.messagesSearch = '/messages-search';
 config.messages.endpoints.messagesBatch = '/messages-batch';
 
+config.messages.scheduler = {};
```

Comment:
`scheduler` isn't a good name for this. We should make it more specific to the job that's running.